packages/common/src/fs.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs/promises'
+import { isErrnoException } from './util'
+
+export const fileExists = async (location: string): Promise<boolean> => {
+  try {
+    await fs.access(location, fs.constants.F_OK)
```

Comment:
`exists` is deprecated & `existsSync` is well... synchronous. 

`access` is lighterweight than `stat` & gives us what we want

Reply:
Ace!